projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.8-beta - 2022-03-29
+### Enhancements
+- Add back the upgrade nudge to the Payment Button block.
+- Payment Blocks: Added links to manage subscriptions and fees to the Product Management controls.
+- Adding language attributes to code blocks.
+- Dashboard: add a card to highlight the blocks available in Jetpack.
+- Dashboard: add highlight of the Jetpack blocks to the settings page.
+- Form Block: offer option to install/activate the Jetpack CRM plugin from the Form Block options.
+- Jetpack Blocks: make settings discoverable and toggleable.
+- Payments Block: introduce new Post Publish panel to highlight the options available with Payments blocks.
+- Payments Block now uses the product management controls to manage subscriptions. Migrated the Payments Block to a functional component and cleaned up code.
+- Slideshow shortcode: adds the Jetpack namespace to slideshow shortcode CSS class names.
+
+### Improved compatibility
+- Added support for the `muted`, `controls` and `playsinline` properties on the wpvideo and videopress shortcodes.
+
+### Bug fixes
+- Assistant: make call-to-actions point to checkout page.
```

Comment:
You'll want to prefix each entry with a "category", a feature name. In the case of blocks, I would recommend using a similar format for each one, such as "Payments Block:"